cmd/clef/main.go

```diff
@@ -923,13 +923,13 @@ func testExternalUI(api *core.SignerAPI) {
 		}
 	}
 	expectApprove := func(testcase string, err error) {
-		if err == nil || err == accounts.ErrUnknownAccount {
+		if errors.Is(err, accounts.ErrUnknownAccount) {
```

Comment:
The behavior is changed here. If err == nil, `errors.Is(err, accounts.ErrUnknownAccount)` will be false.

Reply:
> The behavior is changed here. If err == nil, `errors.Is(err, accounts.ErrUnknownAccount)` will be false.

I'm fixed